commit 09fcc652fefdb58e9cb729e8a604607d8805140c
Author: Fujii Masao <fujii@postgresql.org>
Date:   Wed Jul 16 08:32:52 2025 +0900

    doc: Fix confusing description of streaming option in START_REPLICATION.
    
    Previously, the documentation described the streaming option as a boolean,
    which is outdated since it's no longer a boolean as of protocol version 4.
    This could confuse users.
    
    This commit updates the description to remove the "boolean" reference and
    clearly list the valid values for the streaming option.
    
    Back-patch to v16, where the streaming option changed to a non-boolean.
    
    Author: Euler Taveira <euler@eulerto.com>
    Reviewed-by: Fujii Masao <masao.fujii@gmail.com>
    Discussion: https://postgr.es/m/8d21fb98-5c25-4dee-8387-e5a62b01ea7d@app.fastmail.com
    Backpatch-through: 16

diff --git a/doc/src/sgml/protocol.sgml b/doc/src/sgml/protocol.sgml
index 1b38e5d725a..6c14da0cc76 100644
--- a/doc/src/sgml/protocol.sgml
+++ b/doc/src/sgml/protocol.sgml
@@ -3512,15 +3512,17 @@ psql "dbname=postgres replication=database" -c "IDENTIFY_SYSTEM;"
      <term>
       streaming
      </term>
      <listitem>
       <para>
-       Boolean option to enable streaming of in-progress transactions.
-       It accepts an additional value "parallel" to enable sending extra
-       information with some messages to be used for parallelisation.
-       Minimum protocol version 2 is required to turn it on.  Minimum protocol
-       version 4 is required for the "parallel" option.
+       Option to enable streaming of in-progress transactions. Valid values are
+       <literal>off</literal> (the default), <literal>on</literal> and
+       <literal>parallel</literal>. The setting <literal>parallel</literal>
+       enables sending extra information with some messages to be used for
+       parallelization. Minimum protocol version 2 is required to turn it
+       <literal>on</literal>.  Minimum protocol version 4 is required for the
+       <literal>parallel</literal> value.
       </para>
      </listitem>
     </varlistentry>
 
     <varlistentry>
